tsconfig.json

```diff
@@ -20,6 +20,7 @@
     "inlineSources": true,
     "baseUrl": ".",
     "module": "es6",
+    "skipLibCheck": true,
```

Comment:
Why do we need to add this now?

Reply:
`ember-mocha` makes a lot of warnings in the console because of `mocha` typings, it doesn’t impact code at runtime, it just skips the checking when building the app